code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -92,6 +92,13 @@
 /turf/open/return_analyzable_air()
 	return return_air()
 
+/turf/open/proc/has_gas(datum/gas, mole_amount)
```

Comment:
Document this.

Also, this feels like it should be a macro instead.

Reply:
Moved this to `datum/gas_mixture`